pkg/component/observability/opentelemetry/collector/constants/constants.go

```diff
@@ -24,8 +24,10 @@ const (
 	PushEndpoint = "/opentelemetry.proto.collector.logs.v1.LogsService/Export"
 	// PushPort is the port that the OTLP receiver listens on in the OpenTelemetry Collector deployment.
 	PushPort = 4317
-	// KubeRBACProxyPort is the port that the KubeRBACProxy listens on in the OpenTelemetry Collector deployment.
-	KubeRBACProxyPort int32 = 8080
+	// KubeRBACProxyLokiReceiverPort is the port that the KubeRBACProxy that forwards to the `loki` receiver listens on in the OpenTelemetry Collector deployment.
+	KubeRBACProxyLokiReceiverPort int32 = 8081
```

Comment:
We should use the term `vali` instead of `loki`.

From https://github.com/gardener/gardener/pull/7318:
> Note that these forks are no longer using the Grafana and Loki terms / trademarks, hence they have new names. Consequently, we replaced all the occurrences of these strings with the new names also in the Gardener repository.

Suggestion:
```suggestion
	// KubeRBACProxyValiReceiverPort is the port that the KubeRBACProxy that forwards to the `vali` receiver listens on in the OpenTelemetry Collector deployment.
	KubeRBACProxyValiReceiverPort int32 = 8081
```

Reply:
This was actually due to initially this feature having a different implementation. The `rbac-proxy` was going to route to the `loki` receiver in the collector, but this was changed.
Good catch!